osquery/tables/utility/time.cpp

```diff
@@ -56,6 +56,16 @@ QueryData genTime(QueryContext& context) {
   char iso_8601[21] = {0};
   strftime(iso_8601, sizeof(iso_8601), "%FT%TZ", &gmt);
 
+  if (context.isColumnUsed("timestamp_sys100ns")) {
+    FILETIME ft = {0};
```

Comment:
You'll need to either `#ifdef WIN32` this away from posix, or drop it into one of the abstraction classes to prevent posix boxes from trying to build it.